docs/standard/serialization/binaryformatter-security-guide.md

```diff
@@ -1,29 +1,30 @@
 ---
-title: "BinaryFormatter security guide"
-description: This article describes the security risks inherent in the BinaryFormatter type and recommendations for different serializers to use.
-ms.date: "07/11/2020"
+title: "Deserialization risks in use of BinaryFormatter and related types"
+description: Learn about security vulnerabilities of BinaryFormatter, SoapFormatter, NetDataContractSerializer, LosFormatter, and ObjectStateFormatter. The article recommends safer alternatives.
+ms.date: 03/02/2022
 ms.author: levib
 author: GrabYourPitchforks
 ---
-# BinaryFormatter security guide
+# Deserialization risks in use of BinaryFormatter and related types
```

Comment:
I think these changes are probably fine (@GrabYourPitchforks to confirm), but I just wanted to note that the low organic search rate is likely due to the fact that most page views are due to dedicated aka.ms links to this page from the obsolete attribute text: https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/libraries/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.cs#L20. In this case, the percentage may be low, but the actual number of organic search page views may _not_ be low compared to articles with a higher percentage of organic search page views.